src/components/schedule-page/schedule-page.component.tsx

```diff
@@ -73,6 +76,7 @@ export function SchedulePage({ editModeHandler }: SchedulePageOptions): JSX.Elem
               <Route path="/" component={ViewOnly} exact />
             </Switch>
           )}
+          {ParseErrorModal}
```

Comment:
Czy nie lepiej by było umieścić to w samym kontekście? 

Bo teraz ktoś użyje takiego destruct'a
`const {handleImport, ParseModel} = useImportModal()`

i da ParseModal w swoim komponencie. 
I przy wczytywaniu modala on będzie pojawiać się w dwóch miejscach

Reply:
W sumie nie wpadłem na to. Spróbuję czy zadziała.

Reply:
Śmiga, fajny pomysł :D